crates/claims/crates/data-integrity/suites/src/suites/w3c/ecdsa_rdfc_2019.rs

```diff
@@ -0,0 +1,139 @@
+//! `ecdsa-rdfc-2019` cryptosuite implementation.
+//!
+//! See: <https://www.w3.org/TR/vc-di-ecdsa/#ecdsa-rdfc-2019>
+use core::fmt;
+
+use k256::sha2::{Sha256, Sha384};
+use ssi_data_integrity_core::{
+    canonicalization::{
+        CanonicalClaimsAndConfiguration, CanonicalizeClaimsAndConfiguration,
+        HashCanonicalClaimsAndConfiguration,
+    },
+    signing::{AlgorithmSelection, AlgorithmSelectionError, Base58Btc, MultibaseSigning},
+    suite::{
+        standard::{HashingAlgorithm, HashingError},
+        NoConfiguration,
+    },
+    ProofConfigurationRef, StandardCryptographicSuite, TypeRef,
+};
+use ssi_verification_methods::Multikey;
+use static_iref::iri;
+
+/// The `ecdsa-rdfc-2019` cryptosuite.
+///
+/// See: <https://www.w3.org/TR/vc-di-ecdsa/#ecdsa-rdfc-2019>
+#[derive(Debug, Default, Clone, Copy)]
+pub struct EcdsaRdfc2019;
+
+impl EcdsaRdfc2019 {
+    pub const NAME: &'static str = "DataIntegrityProof";
+
+    pub const IRI: &'static iref::Iri = iri!("https://w3id.org/security#DataIntegrityProof");
+}
+
+impl StandardCryptographicSuite for EcdsaRdfc2019 {
+    type Configuration = NoConfiguration;
```

Comment:
Should we use `AddProof` for data integrity context v2?

Reply:
I don't think that should be required. Do we need it somewhere?